test/aero_info.go

```diff
@@ -255,7 +260,7 @@ func getZones(ctx goctx.Context, k8sClient client.Client) ([]string, error) {
 	for _, node := range nodes.Items {
 		unqZones[node.Labels["failure-domain.beta.kubernetes.io/zone"]] = 1
 	}
-	var zones []string
+	zones := make([]string, 0, len(unqZones))
```

Comment:
most of the places I am defining the length only, places where we are iterating over map and populating the array, there I am using 0 to avoid handling separate index variable.